attestation/environment/environment.go

```diff
@@ -47,9 +48,65 @@ type EnvironmentAttestor interface {
 }
 
 func init() {
-	attestation.RegisterAttestation(Name, Type, RunType, func() attestation.Attestor {
-		return New()
-	})
+	attestation.RegisterAttestation(Name, Type, RunType, func() attestation.Attestor { return New() })
+
+	// registry.BoolConfigOption(
```

Comment:
sorry if I'm missing some context, is there a reason why this is commented out at the moment @matglas ?

Reply:
Its a WIP. This has to move to cli part to make it work again with flags.

Reply:
This is fixed now in PR https://github.com/in-toto/witness/pull/522